src/reports/library/cell.hpp

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include <memory>
+#include <vector>
+
+class Cell {
+  private:
+    int m_gid;
+    int m_vgid;
```

Comment:
what is a vgid ?

Reply:
virtual gid

Reply:
There is not such a thing in SONATA anymore. 
There is  "model types" property instead, that specifies that it is a virtual node eventually. 
"A model_type attribute allows nodes to be configured as biophysical, point_neuron, etc. and also virtual one may be provided to specify external (or virtual) nodes that are not explicitly simulated but provide inputs to the network."